.github/workflows/unittest.yml

```diff
@@ -4,18 +4,18 @@ on: [push]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why do we need a set version and not latest?

Reply:
Latest isn't, erm, latest. And it suddenly started breaking (python 3.7/3.8 missing from distro) without any relevant changes on our side, and experimentally moving to a later version fixed it.